packaging/examples/metrics/grafana-dashboards/strimzi-operators.json

```diff
@@ -1506,6 +1497,128 @@
         "x": 0,
         "y": 24
       },
+      "id": 59,
+      "panels": [],
+      "title": "Certificates",
+      "type": "row"
+    },
+    {
+      "datasource": "$DS_PROMETHEUS",
+      "fieldConfig": {
+        "defaults": {
+          "custom": {
+            "align": null,
+            "filterable": true
+          },
+          "mappings": [],
+          "thresholds": {
+            "mode": "absolute",
+            "steps": [
+              {
+                "color": "green",
+                "value": null
+              }
+            ]
+          },
+          "unit": "string"
+        },
+        "overrides": [
+          {
+            "matcher": {
+              "id": "byName",
+              "options": "Expiry Time"
+            },
+            "properties": [
+              {
+                "id": "unit",
+                "value": "dateTimeAsIso"
+              }
+            ]
+          },
+          {
+            "matcher": {
+              "id": "byName",
+              "options": "Time To Expiry"
+            },
+            "properties": [
+              {
+                "id": "unit",
+                "value": "dateTimeFromNow"
+              }
+            ]
+          }
+        ]
+      },
+      "gridPos": {
+        "h": 8,
+        "w": 24,
+        "x": 0,
+        "y": 25
+      },
+      "id": 57,
+      "options": {
+        "showHeader": true
+      },
+      "pluginVersion": "7.3.7",
+      "targets": [
+        {
+          "expr": "sort (min by (cluster, type, namespace) (strimzi_certificate_expiration_timestamp_ms))",
```

Comment:
@maciej-tatarski this one is for you

Reply:
@scholzj the problem is that `namespace` label is somewhat special, because prometheus will apply it by default for the `pod` metric was scraped from. Because of that `namespace` label that we are exporting is relabeled to `exported_namespace`. I'm not sure if it would be better to just change the dashboard to use `exported_namespace` or change the label to something like `resource_namespace` or `cluster_namespace`. I've noticed that we are already using `resource_namespace` in `strimzi_resource_state`.

Reply:
Good catch!

Reply:
@maciej-tatarski Not in my homelab right now. But I'm pretty sure my Prometheus was showing the label correctly as `namespace="myproject2"` and yet it did not work. So not sure how exactly it works and where it gets the other label. I think using `resource_namespace` label for this metric is fine as it is a new metric. But I guess we should not change it for any existing metrics as that would break things for users.

As for `exported_namespace` -> if you are sure it works for everyone, I guess we could use it. But again, my Prometheus does not show any such label so I'm not sure where it would appear.

Reply:
I'm pretty sure it works this way, however people can have different than default relabeling settings in prometheus. Let's change the label from `namespace` to `resource_namespace` as we are using it in other metrics anyway f.e in `strimzi_resource_state`.

Reply:
If needed @scholzj, lets have a call of 30 mins to settle it 😄 Except 14-15 CEST, I'm free from 13 until 15:45

Reply:
@steffen-karlsson As I said, I'm fine with `resource_namespace` for a new metric if all existing metrics remain unchanged. That said, do you need to also adjust `KafkaAssemblyOperatorMetricsHolder.removeMetricsForCertificates`? That seems to rely on `MetricsUtils.getAllMetricTags` and `MetricsUtils.getRequiredMetricTags` that uses `namespace`.

Reply:
@scholzj to keep in the rest of the metrics, `namespace` will stay in the cert metrics as well, together with the `resource_namespace`, otherwise it needs a lot not so understandable if-else.

I suggest to go with this now, which is applied, dashboard and alerts is changed and then I'd love to go though the metrics logic in the project and re-do it and make it more modular, if you like 😄 Have some ideas on what to apply, but it will be to large a change for this PR also considering the time open.